src/DataViews/CallstackDataView.cpp

```diff
@@ -129,21 +124,21 @@ std::vector<std::vector<std::string>> CallstackDataView::GetContextMenuWithGroup
     }
   }
 
-  std::vector<std::string> action_group;
+  std::vector<std::string_view> action_group;
   if (enable_load) action_group.emplace_back(kMenuActionLoadSymbols);
   if (enable_select) action_group.emplace_back(kMenuActionSelect);
   if (enable_unselect) action_group.emplace_back(kMenuActionUnselect);
   if (enable_disassembly) action_group.emplace_back(kMenuActionDisassembly);
   if (enable_source_code) action_group.emplace_back(kMenuActionSourceCode);
```

Comment:
If you decided to move from string_view back to string in the function signature, then you would need to make explicit constructions of strings here.

Reply:
Done! Thanks for the suggestion!